### PR TITLE
(S) Get event by ID

### PR DESCRIPTION
**Pull request by cmei4444**
Added ability to retrieve one event by its unique event ID. 

Slightly restructured the way events are converted into dict form so an `event_id` field doesn't get stored in MongoDB, which could have led to duplicate or conflicting info. The `get_dict` function now just moves the info in `event_id` to a field called `_id`.

Namedtuple doesn't allow `_id` as a field so I couldn't just modify the required attributes.

### CODE REVIEW

**Comment by cmei4444** [comment]
> I think there's a problem with your post adding logic. MongoDB defaults unique `ObjectID`s as the type for the `_id` field, but it allows you to override that to be whatever datatype you want as long as it is unique and not an array. This can sometimes be useful, but I think you probably want all event IDs to be unique randomly generated `ObjectID`s. To do this, your uploading function must construct an `ObjectID` from the string parameter that was passed to the request (see [here](https://github.com/knative-portability/large-events/blob/master/posts/app.py)), and your tests need to use valid string representations of `ObjectID`s which are 24-char hex strings (see line 189 [here](https://github.com/knative-portability/large-events/blob/master/posts/test_routes.py)).

Do you mean I should create an ObjectID from the string passed to get_one_event? That seems the most similar to what you did in the posts app.py file.

**Comment by mukobi** [comment]
> Do you mean I should create an ObjectID from the string passed to get_one_event? That seems the most similar to what you did in the posts app.py file.

Yeah, but then you'll find your tests won't work if you are testing with fake ObjectIDs unless they are formatted as 24-char hex strings. Alternatively, I guess you could ignore the `_id` field and generate your own event ID and store it as a string using something like the `uuid` lib.

File: events/app.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import datetime
 import json
 import pymongo
-from bson import json_util
+from bson import json_util, ObjectId
```
Comment (evankanderson) [inline events/app.py]: Where is `ObjectId` used?

Comment (mukobi) [reply events/app.py]: It's for serializing the `_id` field when searching for events by ID. MongoDB doesn't let you search for a field of type ObjectId by string, so you have to create an ObjectId yourself. See line 67.

Comment (evankanderson) [reply events/app.py]: D'oh... I totally missed this while scanning.

Thanks!

**Pull request merged**